reputation/reputation.go

```diff
@@ -83,17 +83,10 @@ func (rm *Module) AddSource(id string, maddr ma.Multiaddr) error {
 
 // QueryMiners makes a filtered query on the scored-sorted miner list.
 // Empty filter slices represent no-filters applied.
-func (rm *Module) QueryMiners(n int, excludedMiners []string, countryCodes []string, trustedMiners []string) ([]MinerScore, error) {
-	if n < 1 {
-		return nil, fmt.Errorf("the number of miners should be greater than zero")
-	}
-
+func (rm *Module) QueryMiners(excludedMiners []string, countryCodes []string, trustedMiners []string) ([]MinerScore, error) {
```

Comment:
Like I said, this no longer takes a number of miners to find. It just returns all miners that match the provided filters.